evalml/model_understanding/prediction_explanations/_algorithms.py

```diff
@@ -69,11 +67,11 @@ def _compute_shap_values(pipeline, features, training_data=None):
         if ws:
             logger.debug(f"_compute_shap_values TreeExplainer: {ws[0].message}")
         shap_values = explainer.shap_values(features, check_additivity=False)
-        # shap only outputs values for positive class for Catboost binary estimators.
+        # shap only outputs values for positive class for Catboost/Xgboost binary estimators.
         # this modifies the output to match the output format of other binary estimators.
         # Ok to fill values of negative class with zeros since the negative class will get dropped
         # in the UI anyways.
-        if estimator.model_family == ModelFamily.CATBOOST and pipeline.problem_type == ProblemTypes.BINARY:
+        if estimator.model_family in {ModelFamily.CATBOOST, ModelFamily.XGBOOST} and pipeline.problem_type == ProblemTypes.BINARY:
```

Comment:
@freddyaboulton no action needed here in this PR but: this change reminds me of some work we'll need to do eventually for timeseries. There are a lot of places where we do
```python
pipeline.problem_type == ProblemTypes.BINARY
```
instead of
```python
is_binary(pipeline.problem_type)
```
which will catch time series binary as well

Reply:
Great call @dsherry !